v22.1/aws-dms.md

```diff
@@ -5,27 +5,33 @@ toc: true
 docs_area: develop
 ---
 
-This page has instructions for setting up [AWS DMS](https://aws.amazon.com/dms/) to migrate data to CockroachDB from an existing, publicly-hosted database containing application data such as MySQL, Oracle, or PostgreSQL.
+This page has instructions for setting up [AWS Database Migration Service (DMS)](https://aws.amazon.com/dms/) to migrate data to CockroachDB from an existing, publicly-hosted database containing application data such as MySQL, Oracle, or PostgreSQL.
```

Comment:
No need to hyphenate adverbs ending in "-ly", can open up as "publicly hosted database"